lib/oidcstrategy.js

```diff
@@ -359,36 +359,34 @@ function Strategy(options, verify) {
   if (options.isB2C !== true)
     options.isB2C = false;
 
-  // common endpoint is not allowed for B2C
-  if (options.isCommonEndpoint && options.isB2C) {
-    throw new Error(`Cannot use common endpoint for B2C. Please use your tenant name or tenant guid.`);
-  }
+  // common endpoint will not work for B2C unless they provide the tenant name or tenant guid in 'passport.authenticate'
+  if (options.isCommonEndpoint && options.isB2C)
+    log.info(`In order to use common endpoint for B2C, please provide your tenant name or tenant guid when you call 'passport.authenticate'.`);
 
```

Comment:
should we add check for tenant name/giud  and than throw error ?
 is it kind of advise ? it will be written to log for the case when tenant name/giud  provided and not provided, so it does not clearly reflect execution  of ther program